site/_docs/howto.md

```diff
@@ -474,7 +474,7 @@ $ ./src/main/scripts/generate-protobuf.sh
 The following sections are of interest to Calcite committers and in
 particular release managers.
 
-## Managing Calcite repositories through GitHub (for Calcite committers)
+## Managing Calcite repositories through GitHub
 
 Committers have write access to Calcite's 
 [ASF git repositories](https://gitbox.apache.org/repos/asf#calcite) hosting
```

Comment:
Why should we remove the suffix?

Reply:
The whole section is about committers (see line 472) and the suffix is repeated in every subsection so IMHO it is redundant. It makes headers and links longer without adding new information.

Reply:
> The whole section is about committers (see line 472) and the suffix is repeated in every subsection so IMHO it is redundant. It makes headers and links longer without adding new information.

-1, I'm not convinced.

Reply:
I think @zabetak has a point. If we have a section called "Advanced topics for committers", then it seems redundant to have every subsection in there called "XYZ (for Calcite committers)".
Right before that we have an "Advanced topics for developers" section and we do not have the suffix "(for developers)" on each subsection.

Reply:
> The whole section is about committers (see line 472) and the suffix is repeated in every subsection so IMHO it is redundant. It makes headers and links longer without adding new information.

Makes sense. Thanks for your explanation.